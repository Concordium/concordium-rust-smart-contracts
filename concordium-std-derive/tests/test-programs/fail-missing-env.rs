//! Ensure that the macros generate compilable code.

use concordium_contracts_common::*;
use concordium_std_derive::*;

const ACC: AccountAddress = account_address_env!("TEST");

fn f() {
    println!("{:?}", ACC.to_string());
}

fn main() {}
