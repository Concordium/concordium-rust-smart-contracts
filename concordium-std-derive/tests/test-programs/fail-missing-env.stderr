error: Environment variable error: NotPresent
 --> tests/test-programs/fail-missing-env.rs:6:50
  |
6 | const ACC: AccountAddress = account_address_env!("TEST");
  |                                                  ^^^^^^
